stable/coscale/values.yaml

```diff
@@ -0,0 +1,24 @@
+# Default values for CoScale Helm package.
+
+image:
+    repository: "coscale/coscale-agent"
+    tag: "latest"
+    pullPolicy: "Always"
+
+coscale:
+    # Required: You need a CoScale AppId before running agents.
+    AppId: ""
```

Comment:
Variables should start with a lowercase letter. See https://github.com/kubernetes/helm/blob/master/docs/chart_best_practices/values.md#naming-conventions